chef_master/source/resource_remote_directory.rst

```diff
@@ -197,78 +158,163 @@ The remote_directory resource has the following properties:
    Purge extra files found in the target directory.
 
 ``recursive``
-   **Ruby Type:** true, false
+   **Ruby Type:** true, false | **Default Value:** ``true``
+
+   Create or delete directories recursively. The chef-client must be able to create the directory structure, including parent directories (if missing), as defined in ``COOKBOOK_NAME/files/default/REMOTE_DIRECTORY``.
+
+``rights``
+   **Ruby Type:** Integer, String
+
+   Microsoft Windows only. The permissions for users and groups in a Microsoft Windows environment. For example: ``rights <permissions>, <principal>, <options>`` where ``<permissions>`` specifies the rights granted to the principal, ``<principal>`` is the group or user name, and ``<options>`` is a Hash with one (or more) advanced rights options.
+
+``source``
+   **Ruby Type:** String | **Default Value:** ``The base portion of the 'path' property. For example '/some/path/' would be 'path'.``
```

Comment:
Would it be possible to shorten this Default Value? This may require manually editing the description.

Reply:
@tas50 ^

Reply:
@tas50 I went ahead and fixed it.